src/providers/Profile.js

```diff
@@ -38,6 +39,11 @@ function ProfileProvider({ children }) {
         setBox(box)
       }
     } catch (err) {
+      setProfile(profile => ({
+        ...profile,
+        onboardingSkipped: true,
```

Comment:
Would rename to `confirmationFailed` because it's not something you can skip, if you don't confirm the modal you won't be authenticated.